edb/edgeql/codegen.py

```diff
@@ -1659,10 +1659,10 @@ def visit_ReleaseSavepoint(self, node: qlast.ReleaseSavepoint) -> None:
 
     def visit_DescribeStmt(self, node: qlast.DescribeStmt) -> None:
         self.write(f'DESCRIBE ')
-        if node.object:
-            self.visit(node.object)
+        if isinstance(node.object, qlast.DescribeGlobal):
+            self.write(str(node.object))
```

Comment:
Please add `DescribeGlobal.to_edgeql()` method like in enums in `edb/edgeql/qltypes.py`. Makes things a little clearer in places like this.